samples/react/README.md

```diff
@@ -0,0 +1,85 @@
+This project integrates monaco-kusto in a create-react-app application.
+
+The following changes were made to a default create-react-app application:
+
+1. Used script tags to import Kusto language service (this piece of code is C# code transpiled to javascript. it has several issues that prevent it from being imported as a regular import statement).
+
+2. Added a prepare task to package.json that copies over monaco and monaco-kusto to the public folder (this is done because we're using UMD to require monaco and monaco-kusto in runtime. otherwise, we would have to eject create-react-app and play with webpack config as explained [here](https://github.com/microsoft/monaco-editor/blob/master/docs/integrate-esm.md) )
+
+3. Added monaco-kusto.js which takes care of loading monaco (by using @monaco-editor/react package) and then loading monaco-kusto
```

Comment:
Reading the ReadMe in https://www.npmjs.com/package/@monaco-editor/react and looking at their [Examples](https://monaco-react.surenatoyan.com/), I can see that this library supports downloading monaco from a CDN. For this sample, it might be simpler to let @monaco-editor/react to download monaco from the CDN.